.eslintrc.json

```diff
@@ -0,0 +1,19 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": "eslint:recommended",
+    "parserOptions": {
+        "ecmaVersion": 12,
+        "sourceType": "module"
+    },
+    "rules": {
+       "no-undef":"off",
+       "no-unused-vars": ["error", { "argsIgnorePattern": "req|res|next|val" }],
```

Comment:
Why are these rules turned off?

Reply:
It is because when I call `process.env.PORT` they tell me `process` is undefined

Reply:
There's another way to deal with that read more here: https://eslint.org/docs/rules/no-process-env